hbase-server/src/main/java/org/apache/hadoop/hbase/constraint/package-info.java

```diff
@@ -18,110 +18,121 @@
 
 /**
  * Restrict the domain of a data attribute, often times to fulfill business rules/requirements.
+ * <h2>Table of Contents</h2>
+ * <ul>
+ * <li><a href="#overview">Overview</a></li>
+ * <li><a href="#concurrency">Concurrency and Atomicity</a></li>
+ * <li><a href="#caveats">Caveats</a></li>
+ * <li><a href="#usage">Example Usage</a></li>
+ * </ul>
+ * <h2><a name="overview">Overview</a></h2> Constraints are used to enforce business rules in a
+ * database. By checking all {@link org.apache.hadoop.hbase.client.Put Puts} on a given table, you
+ * can enforce very specific data policies. For instance, you can ensure that a certain column
+ * family-column qualifier pair always has a value between 1 and 10. Otherwise, the
+ * {@link org.apache.hadoop.hbase.client.Put} is rejected and the data integrity is maintained.
+ * <p/>
+ * Constraints are designed to be configurable, so a constraint can be used across different tables,
+ * but implement different behavior depending on the specific configuration given to that
+ * constraint.
+ * <p/>
+ * By adding a constraint to a table (see <a href="#usage">Example Usage</a>), constraints will
+ * automatically enabled. You also then have the option of to disable (just 'turn off') or remove
```

Comment:
missing 'be'?

Reply:
I haven't changed the wording, just reformat. Let me fix.